PowerEditor/src/Parameters.cpp

```diff
@@ -653,6 +653,7 @@ void cutString(const TCHAR* str2cut, vector<generic_string>& patternVect)
 {

```

Comment:
`cutString()` is a generic function which uses white space to build token array.
Not mention that this function is used in other places,  adding other semantic here for the specific need here is just wrong.

Reply:
Note that I didn't look at code for this PR other than what is shown in github's web view.
I did run the code from Appveyor build and it worked, but obviously that is only part of what is required for successful PR.
I'm sorry for not looking deeper at it.  I fully agree with rejection reason.

Reply:
My apologies for choosing cutString as the place for the change.  I initially put the code in FindReplaceDlg but thought it might cut down on code repetition if it were in cutString.  I'll try to do a better job of analyzing the code in the future.

Reply:
@eljefe7000 Live and learn, like always...everyone...every day...